.github/scripts/.helm-tests/default/result.yaml

```diff
@@ -158,135 +158,162 @@ spec:
+                          - name
+                        type: object
+                      value:
+                        description: Value is the value the provider returned
+                        type: string
+                    type: object
+                  description: StoredValues contains all analysis values that have already
+                    been retrieved successfully
+                  type: object
+                timeframe:
+                  description: Timeframe describes the time frame which is evaluated
+                    by the Analysis
                   properties:
-                    errMsg:
-                      description: ErrMsg stores any possible error at retrieval time
+                    from:
+                      description: From is the time of start for the query, this field
+                        follows RFC3339 time format
+                      format: date-time
                       type: string
-                    objectiveReference:
-                      description: Objective store reference to corresponding objective
-                        template
-                      properties:
-                        name:
-                          description: Name defines the name of the referenced object
-                          type: string
-                        namespace:
-                          description: Namespace defines the namespace of the referenced
-                            object
-                          type: string
-                      required:
-                      - name
-                      type: object
-                    value:
-                      description: Value is the value the provider returned
+                    recent:
+                      description: Recent describes a recent timeframe using a duration
+                        string. E.g. Setting this to '5m' will cause the Analysis to
+                        be performed for the last five minutes
+                      pattern: ^0|([0-9]+(\.[0-9]+)?(ns|us|µs|ms|s|m|h))+$
+                      type: string
+                    to:
+                      description: To is the time of end for the query, this field follows
+                        RFC3339 time format
+                      format: date-time
                       type: string
                   type: object
-                description: StoredValues contains all analysis values that have already
-                  been retrieved successfully
-                type: object
-              warning:
-                description: Warning returns whether the analysis returned a warning
-                type: boolean
-            required:
-            - state
-            type: object
-        type: object
-    served: true
-    storage: true
-    subresources:
-      status: {}
+                warning:
+                  description: Warning returns whether the analysis returned a warning
```

Comment:
```suggestion
                  description: Warning is set to `true` if  the analysis returned a warning
```

Reply:
we can improve this in a separate PR, as this one is about the new time frame selector